src/pages/ResultsListPage.js

```diff
@@ -149,7 +149,7 @@ class ResultsListPage extends BaseResultsPage {
                         object={object}
                       />;
 
-                    var klass = elem.displayName || "other";
+                    var klass = elem.type.displayName || "other";
```

Comment:
`elem.displayName` never worked - it always just went to `"other"`, both in dev and prod.